cudaextender/src/ungapped_xdrop.cu

```diff
@@ -76,13 +76,6 @@ UngappedXDrop::UngappedXDrop(const int32_t* h_score_mat, const int32_t score_mat
     //Figure out memory requirements for cub functions
     size_t temp_storage_bytes = 0;
```

Comment:
Thanks for making this change. Please feel free to merge this after removing the temp_storage_bytes unused var.